### PR TITLE
Move hardcoded values to variables.

### PR DESCRIPTION
**Pull request by janisz**
## High-level description

This change allow customizing terraform with variables.

## Corresponding DC/OS tickets (required)

  - [D2IQ-ID](https://jira.d2iq.com/browse/D2IQ-ID) JIRA title / short description.

## Related tickets (optional)

  - [D2IQ-ID](https://jira.mesosphere.com/browse/D2IQ-<number>) JIRA title / short description.

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by janisz** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by janisz** [comment]
@mesosphere-mergebot changelog-not-required internal change

**Comment by janisz** [comment]
@mesosphere-mergebot help

**Comment by janisz** [comment]
@mesosphere-mergebot request-review @jongiddy  @jeschkies  @DominikDary

File: test_util/main.tf [hunk 1]
```diff
@@ -74,7 +85,7 @@ module "dcos" {
 
   dcos_variant              = "${var.variant}"
   dcos_version              = "2.1.0-beta1"
-  dcos_license_key_contents = "${file("./license.txt")}"
+  dcos_license_key_contents = "${file("${var.license_file}")}"
```
Comment (jeschkies) [inline test_util/main.tf]: @fatz, should this be a file path or should we rather use the content and remove `file`?

Comment (fatz) [reply test_util/main.tf]: so from CI perspective plain text is easier to filter from output. Just thinking of Jenkins secret text vs secret file

File: test_util/main.tf [hunk 2]
```diff
@@ -33,6 +33,17 @@ variable "windowsagent_num" {
   description = "Defines the number of Windows agents for the cluster."
 }
 
+variable "ssh_public_key_file" {
+  type = "string"
+  default = "./tf-dcos-rsa.pem.pub"
```
Comment (jeschkies) [inline test_util/main.tf]: This is not a good default IMHO. It should be `~/.ssh/id_rsa.pub`. Remember that we want to use the `main.tf` locally as well.

Comment (janisz) [reply test_util/main.tf]: That's why I allow configuration. I'd not like to break CI

Comment (jeschkies) [reply test_util/main.tf]: Ok. I'll change it in the Makefile then.

**Comment by janisz** [comment]
@fatz @jeschkies I updated the PR

**Comment by janisz** [comment]
@jeschkies What exactly should I change in the readme? All variables I set are internal changes that were already exposed to Makefile user?

**Comment by jeschkies** [comment]
@janisz, you a right. I mixed up `DCOS_LICENSE_CONTENTS` and thought it's the path not the content.

**Comment by janisz** [comment]
@mesosphere-mergebot sync

**Comment by janisz** [comment]
:wave: @jeschkies @DominikDary @orsenthil @jongiddy   Can you review, please?

File: test_util/README.md [hunk 3]
```diff
@@ -43,7 +43,9 @@ version. It can be used to test pull request clusters: `TF_VAR_custom_dcos_downl
 `TF_VAR_custom_dcos_download_path_win` define the download for `dcos_generate_config_win.sh`. It defaults to the master
 version. It can be used to test pull request clusters: `TF_VAR_custom_dcos_download_path_win=https://downloads.dcos.io/dcos/testing/pull/6956/windows/dcos_generate_config_win.sh`.
 
-`TF_VAR_variant` determines wether the cluster is `open`, `permissive` or `strict`. It defaults to `open`.
+`TF_VAR_variant` determines wether the cluster is `open` or `ee`. It defaults to `open`.
+
+`TF_VAR_dcos_security` determines wether the cluster is `permissive`, `strict` (only in enterprise) or empty. It defaults to empty string.
```
Comment (jeschkies) [inline test_util/README.md]: Hm, which entity is setting this parameter. Should it be done in TeamCity?

**Comment by janisz** [comment]
@mesosphere-mergebot changelog-not-required everything in readme (only internal chagne)

**Comment by janisz** [comment]
@mesosphere-mergebot sync

**Pull request merged**